CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ if(POLICY CMP0071)
 endif()
 
 project(mixxx VERSION 2.3.0)
+set(MIXXX_VERSION_PRERELEASE "beta") # set to "alpha-pre" "beta" or ""
```

Comment:
```suggestion
set(MIXXX_VERSION_PRERELEASE "beta") # set to "alpha" "beta" or ""
```
?

Reply:
Please add a comment that this is necessary due to this CMake issue: https://gitlab.kitware.com/cmake/cmake/-/issues/16716

Reply:
currently our alpha is called alpha-pre. 
https://github.com/mixxxdj/mixxx/blob/9e974cdb440ce931f70a89005060f810705c6f89/src/_version.h#L1

I think it is reasonable to change this along with the other changes from git describe.